src/ldp/http/BasicRequestParser.ts

```diff
@@ -37,6 +40,7 @@ export class BasicRequestParser extends RequestParser {
 
   public async handle(input: HttpRequest): Promise<Operation> {
     if (!input.method) {
+      this.logger.error('Missing method in HttpRequest.');
```

Comment:
```suggestion
```

@joachimvh ?

Reply:
Plus similar for all cases below where an exact error is thrown already.

Reply:
Depends on what we expect logging to be used for. The person getting the error might not be the one that actually cares that the server is throwing this error. And if this one gets thrown it means something went horribly wrong.

Reply:
I just essentially see the same lines twice now; that can't be a good idea, right? @rubensworks?

Reply:
We can create a `logAndThrow` function that takes an error as input, logs it message and then throws it, but that would obfuscate when we're throwing errors.

Or we could not log when throwing, but then I don't think we'll ever need `logger.error` since we're always throwing in those cases.

Reply:
Any consensus on this already?

Reply:
If both statements remain I would eventually suggest declaring the common error message above it so that it is not duplicated;
```TypeScript
const errMsg = ...
this.logger.warn(errMsg);
throw new ...Error(errMsg);
```

But I will wait for a decision before I change this.

Reply:
Another idea is `throw this.logger.error(new Error('Missing method.'))`, where the `throw` is still explicit.
But let's leave things the way they are for now.